schema/Query.graphql

```diff
@@ -28,4 +28,14 @@ type Query {
     Generate challenge to authenticate with WebAuthn
     """
     generateAuthenticatorChallenge(username: String!): AuthenticationWithWebPublicKeyCredential
+
+    """
+    Get specific life by its ID
+    """
+    getLife(id: ObjectID!): Life
+
+    """
+    List lives
+    """
+    listLives: [Life!]
```

Comment:
the array should be mandatory as well, we expect an empty list if there's nothing

Reply:
Fixed. Will include in next commit (part 5)